haero/constants.hpp

```diff
@@ -60,6 +60,9 @@ static constexpr Real molec_weight_h2so4 = 0.098079;
 ///  Accommodation coefficient (for condensation) value for h2so4 []
 static constexpr Real accom_coef_h2so4 = 0.65;
 
+/// Molecular diffusion volume of h2so4 [-]
```

Comment:
I wonder if this means we need a `molecular_diffusion_constant` field for the `GasSpecies` type. What do you think?

Reply:
Not needed for this PR, though. I'll create an issue to track this question.

Reply:
If we ever need to add the molecular_diffusion for the other gas species then we should add it as a GasSpecies field.  But for now I only used this one for h2so4 and don't have a value to set for the other gasses.